src/Umbraco.Cms.Integrations.Script.Cookiebot/Constants.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace Umbraco.Cms.Integrations.Script.Cookiebot
+{
+    public class Constants
+    {
+        public const string SettingsPath = "Umbraco:Cms:Integrations:Script:Cookiebot:Settings";
```

Comment:
Given we're considering this as a template for other partners to copy, maybe configuration is best outside of `Umbraco:CMS`.  Just in case we were to clash in future.

So better perhaps:

```
Umbraco:Cookiebot:Settings
```